agent/bench-scripts/tests/pbench-specjbb2005/test-57.txt

```diff
@@ -0,0 +1,96 @@
+/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/pbench-postprocess-tools --group=default --dir=/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-57_1900.01.01T00.00.00/3-warehouses:23/sample1
+/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/pbench-send-tools --group=default --dir=/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-57_1900.01.01T00.00.00/1-warehouses:13/sample1
+/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/pbench-send-tools --group=default --dir=/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-57_1900.01.01T00.00.00/2-warehouses:18/sample1
+/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/pbench-send-tools --group=default --dir=/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-57_1900.01.01T00.00.00/3-warehouses:23/sample1
+/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/pbench-start-tools --group=default --dir=/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-57_1900.01.01T00.00.00/1-default/sample1
+/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/pbench-start-tools --group=default --dir=/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-57_1900.01.01T00.00.00/2-default/sample1
+/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/pbench-start-tools --group=default --dir=/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-57_1900.01.01T00.00.00/3-default/sample1
+/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/pbench-stop-tools --group=default --dir=/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-57_1900.01.01T00.00.00/1-default/sample1
+/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/pbench-stop-tools --group=default --dir=/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-57_1900.01.01T00.00.00/2-default/sample1
+/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/pbench-stop-tools --group=default --dir=/var/tmp/pbench-test-bench/pbench-agent/specjbb2005_test-57_1900.01.01T00.00.00/3-default/sample1
+/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/pbench-tool-meister-start --sysinfo=default default
+/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/pbench-tool-meister-stop --sysinfo=default default
+/var/tmp/pbench-test-bench/opt/pbench-agent/unittest-scripts/sleep 5
+--- test-execution.log file contents
++++ specjbb2005_test-57_1900.01.01T00.00.00/metadata.log file contents
+[pbench]
+
+[iterations/1-warehouses:13]
+iteration_number = 1
+iteration_name = 1-warehouses:13
+user_script = specjbb2005
+
+[iterations/2-warehouses:18]
+iteration_number = 2
+iteration_name = 2-warehouses:18
+user_script = specjbb2005
+
+[iterations/3-warehouses:23]
+iteration_number = 3
+iteration_name = 3-warehouses:23
+user_script = specjbb2005
```

Comment:
So...when going from 18 to 23 by fives, the "23" is inclusive?  (Should we have a test like 18-_22_ by fives?)

Reply:
I don't know exactly how `specjbb2005` handles the parameters, but what I wanted for this test was to have the number of JVMs be not 1 so that the logic of  single vs multiple JVMs in `pbench-specjbb2005` would be exercised.  I then used the start/stop/inc arguments to use a different sequence than the previous test.

Reply:
Yup, I understand and support that.  I'm just wondering whether the mock should be running twice instead of thrice.